src/Connection.php

```diff
@@ -60,6 +61,8 @@ public function __construct($hostname, $port, $connectTimeout = null, $connectPe
      * Sets a manually created socket, used for unit testing.
      *
      * @param Socket $socket
+     *
+     * @return $this
```

Comment:
If `return $this` is the correct way to express a method returning its receiver, you could replace `@chainable` here and throughout `src/PheanstalkInterface.php`. As far as I can remember I made up `@chainable` and it's not used by any tools…?